packages/core/src/external/opensearch/lexical/index-create.sh

```
@@ -0,0 +1,48 @@
+# That's not to actually be used directly, but via OS' console or Postman.
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)